docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena 1.2 provides **experimental** support for managing persistent service data using Docker volumes. The exact details of how these Kontena volumes are managed may still change as the implementation evolves.
+
+Volumes are supported as first-class objects in the Kontena Master API, which can be referred to from stack YAML definitions.
+
+Kontena tracks the available Docker volume drivers on each host node, as shown in kontena node show, and will only deploy services using such a volume driver onto a host node that provides that volume driver. In order to use any other volume drivers than the default local driver, those volume drivers must be provided by Docker plugins on the host nodes. Kontena itself does not yet support provisioning Docker plugins onto the grid's host nodes.
+
+
```

Comment:
This needs some general explanation of the relationship between "Volumes" and "Volume Instances", like

> A given Kontena volume can be used by multiple Kontena service instances deployed to different host nodes, and the Kontena scheduler will automatically create multiple separate volume *instances*  for each Kontena volume. These volume instances correspond to a specific Docker volume on a specific host node. The scheduling behavior of each Kontena volume depends on the Volume's `scope`, and any pre-existing volume instances.